tasks/section2.yml

```diff
@@ -385,7 +411,7 @@
       state: stopped
       enabled: no
   when:
-      - rhel7cis_nfs_rpc_server == false and nfs_service_status.stdout == "loaded"
```

Comment:
What is the reason the conditional for `nfs_service_status.stdout == "loaded"` was removed from this task?

Reply:
Idempotency

Reply:
How does removing the check for status = loaded ensure idempotency specifically? In a later commit I noticed that you had to add `changed_when: no` to one of these tasks because it was failing the idempotency checks...I don't want to add `changed_when: no` to a task unless it is `command`, `shell` etc that doesn't properly report state.

Reply:
```molecule idempotence``` does this testing. It tests idempotency of the role by running it a second time having this in ```ansible.cfg```:

```
[defaults]
stdout_callback = actionable
```